gcovr/formats/gcov/read.py

```diff
@@ -91,11 +105,13 @@ def read_report(options: Options) -> CovData:
 
 
 def find_existing_gcov_files(search_path, exclude_dirs):
-    """Find .gcov files under the given search path."""
+    """Find .gcov and .json.gz files under the given search path."""
     LOGGER.debug(f"Scanning directory {search_path} for gcov files...")
     gcov_files = list(
         search_file(
-            re.compile(r".*\.gcov$").match, search_path, exclude_dirs=exclude_dirs
+            re.compile(r".*(\.gcov|\.json\.gz)$").match,
```

Comment:
The `.json.gz` extension seems rather generic, there's a risk of matching other files unrelated to gcov.

The documentation of `--json-format` says it produces files with extension `.gcov.json.gz` so maybe you can use that instead?